server/lib/transferwise.ts

```diff
@@ -44,9 +44,10 @@ export const createQuote = async (
     });
     return getData(response);
   } catch (e) {
-    const message = `Unable to create quote: ${getAxiosError(e)}`;
-    logger.error(message, data);
-    throw new Error(message);
+    logger.error(`Unable to create quote: ${getAxiosError(e)}`, data);
+    const publicMessage =
+      get(e, 'response.data.errors[0].message') || `Sorry, we can't make transfers to ${targetCurrency}.`;
```

Comment:
I think we discussed this before but I don't remember the conclusion, are we 100% sure that the error message from transferwise can't include any sensitive information?

Reply:
Since errors are not being documented by TransferWise, we're not 100% sure.
By what was experienced during development, although the error contains metadata related to the request, the messages are usually generic.
I'll make sure to throw generic errors on our side, at least for the methods that are being used to generate the form.